static/js/containers/SubscriptionsSidebar.js

```diff
@@ -0,0 +1,55 @@
+// @flow
+import React from "react"
+import { connect } from "react-redux"
+import { Link } from "react-router-dom"
+import type { Dispatch } from "redux"
+
+import { channelURL } from "../lib/url"
+import { actions } from "../actions"
+import type { RestState } from "../flow/restTypes"
+import type { Channel } from "../flow/discussionTypes"
+
+type SubscriptionsSidebarProps = {
+  subscriptions: RestState<Array<Channel>>,
+  dispatch: Dispatch
+}
+
+class SubscriptionsSidebar extends React.Component {
+  props: SubscriptionsSidebarProps
+
+  componentWillMount() {
+    const { subscriptions, dispatch } = this.props
+    if (!subscriptions.loaded && !subscriptions.processing) {
+      dispatch(actions.subscriptions.get())
+    }
+  }
+
+  render() {
+    const { subscriptions } = this.props
+
+    if (!subscriptions.data) {
+      return null
+    }
+
+    return (
+      <div className="subscriptions">
+        {subscriptions.data.map(channel =>
+          <div key={channel.name}>
+            <Link to={channelURL(channel.name)}>
+              {channel.title}
+            </Link>
+          </div>
+        )}
+      </div>
+    )
+  }
+}
+
+const mapStateToProps = state => {
+  return {
+    channels:      state.channels,
+    subscriptions: state.subscriptions
+  }
+}
+
+export default connect(mapStateToProps)(SubscriptionsSidebar)
```

Comment:
The channels from `/api/v0/channels/` are the channels you're subscribed to, but not necessarily all the ones you have permission to see, so there is a need for a channel detail view. That's also why I named it `subscriptions`. Regarding the container rendering in another container, I wanted to do it this way since the sidebar is used in both `HomePage` and `ChannelPage`. I can go either way though, maybe we should talk with @aliceriot and @rhysyngsun when they're back in the office